10/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:11.12.0-stretch 
+FROM node:10.16.3-stretch 
```

Comment:
In this correct? version 10 should it not be 11 according the file name, or vice versa?

Reply:
Ignoring 11 for now since it's not the `default`, `stable`, `latest` or `lts`
```
$ nvm list
.....
default -> node (-> v12.9.0)
node -> stable (-> v12.9.0) (default)
stable -> 12.9 (-> v12.9.0) (default)
.....
lts/* -> lts/dubnium (-> v10.16.3)
.....
```